src/quick-view.scss

```diff
@@ -0,0 +1,93 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-quick-view;
+
+$fd-quick-view-avatar-indent: 0.75rem;
+$fd-quick-view-group-item-indent: 0.75rem;
+
+.#{$block} {
+  @include fd-reset();
+
+  @include fd-rtl() {
+    .#{$fd-namespace}-bar {
+      .#{$fd-namespace}-avatar {
+        margin-right: 0;
+        margin-left: $fd-quick-view-avatar-indent;
+      }
+    }
+  }
```

Comment:
Could you move it to `42` line, so it will be 
```
      .#{$fd-namespace}-avatar {
            margin-right: $fd-quick-view-avatar-indent;
           @include fd-rtl() { ....  }
      }
```

Reply:
The implemented version, in my opinion, has more common selectors
Implemented:
```
[dir="rtl"] .fd-quick-view .fd-bar--header-with-subheader .fd-avatar,
.fd-quick-view[dir="rtl"] .fd-bar--header-with-subheader .fd-avatar {...}
```

Suggested:
```
[dir="rtl"] .fd-quick-view .fd-bar--header-with-subheader .fd-avatar,
.fd-quick-view .fd-bar--header-with-subheader .fd-avatar[dir="rtl"] {...}
```
Just I see no use in `.fd-avatar[dir="rtl"]`

But I think now that it could be better to be change bar selector to be more specific:
```
[dir="rtl"] .fd-quick-view .fd-bar--header-with-subheader .fd-avatar,
.fd-quick-view[dir="rtl"] .fd-bar--header-with-subheader .fd-avatar {...}
```